plugins/@grouparoo/sqlite/src/lib/sqlite.ts

```diff
@@ -28,15 +29,22 @@ export class SQLite extends Database {
     });
   }
 
-  async asyncQuery(query: string): Promise<any[]> {
+  async asyncQuery(
+    query: string,
+    params?: SQLiteQueryParamValue[] | Record<string, SQLiteQueryParamValue>
+  ): Promise<Record<string, any>[]> {
```

Comment:
Maybe asyncQuery should take an instance of QueryBuilder, rather than the string query?

Reply:
I looked into this and there's a couple of issues:

1. It's also being used in the test utils. I could switch everything to use the query builder but then I'm coupling the builder with the connection
2. It's unusual to pass a builder as a parameter. The convention is generally to build the output and return the result.

One thought I had was to change the signature so that it's `asyncQuery({ query, params})` but then all simple calls would need to be mad as `asyncQuery({ query: 'select...' })`